src/test/java/plugins/DashboardViewPluginTest.java

```diff
@@ -27,4 +32,82 @@ public void configure_dashboard() {
         j.getLastBuild().shouldSucceed();
 
     }
+
+    @Test
+    public void buildStatisticsPortlet_successNr() {
+        DashboardView v = jenkins.views.create(DashboardView.class, "dashboard");
+        v.configure();
+        v.matchAllJobs();
+
+        BuildStatisticsPortlet stats = v.addBottomPortlet(BuildStatisticsPortlet.class);
+
+        v.save();
+
+        buildSuccessfulJob(createFreeStyleJob());
+
+        v.open();
+
+        assertThat(stats.getNumberOfBuilds(Jobtype.SUCCESS), is(1));
+    }
+
+    @Test
+    public void buildStatisticsPortlet_successPercentage() {
+        DashboardView v = jenkins.views.create(DashboardView.class, "dashboard");
+        v.configure();
+        v.matchAllJobs();
+
+        BuildStatisticsPortlet stats = v.addBottomPortlet(BuildStatisticsPortlet.class);
+
+        v.save();
+
```

Comment:
In order to reduce the duplication you can extract 55-61 into a method, then the code looks like this:
```
        DashboardView v = createDashboard();
        BuildStatisticsPortlet stats = v.getBottomPortlet(BuildStatisticsPortlet.class);
```